src/screen.rs

```diff
@@ -563,7 +562,7 @@ impl Screen {
                     error!("command failed to start: {}", content);
                 }
             } else {
-                let shell = env::var("SHELL").unwrap_or("bash".to_owned());
+                let shell = env::var("SHELL").unwrap_or_else(|_| "bash".to_owned());
```

Comment:
Is this a clippy lint? If so, okay.
If not, I'm not sure if it's really quite worth the optimisation

Reply:
clippy